CMakeLists.txt

```diff
@@ -1089,8 +1089,8 @@ include(PCAPTests)
 include(OpenSSLTests)
 include(CheckNameserCompat)
 include(GetArchitecture)
-include(FindKqueue)
 include(FindCAres)
+include(FindKqueue)
```

Comment:
Wonder if this might be worth a comment. The commit description is verbose, so a git annotate would help.